superset-frontend/src/explore/components/ExploreChartHeader/index.jsx

```diff
@@ -248,134 +226,138 @@ export class ExploreChartHeader extends React.PureComponent {
       ),
     );
     return permissions[0].length > 0;
-  }
+  };
 
-  render() {
-    const { user, form_data: formData, slice } = this.props;
-    const {
-      chartStatus,
-      chartUpdateEndTime,
-      chartUpdateStartTime,
-      latestQueryFormData,
-      queriesResponse,
-    } = this.props.chart;
-    // TODO: when will get appropriate design for multi queries use all results and not first only
-    const queryResponse = queriesResponse?.[0];
-    const chartFinished = ['failed', 'rendered', 'success'].includes(
-      this.props.chart.chartStatus,
-    );
-    return (
-      <StyledHeader id="slice-header" className="panel-title-large">
-        <div className="title-panel">
-          {slice?.certified_by && (
-            <>
-              <CertifiedBadge
-                certifiedBy={slice.certified_by}
-                details={slice.certification_details}
-              />{' '}
-            </>
-          )}
-          <EditableTitle
-            title={this.getSliceName()}
-            canEdit={
-              !this.props.slice ||
-              this.props.can_overwrite ||
-              (this.props.slice?.owners || []).includes(
-                this.props?.user?.userId,
-              )
-            }
-            onSaveTitle={this.props.actions.updateChartTitle}
-          />
+  useEffect(() => {
+    const { userId } = user;
```

Comment:
This will probably fail with an anonymous user

Reply:
thank you! I've removed the destructuring.